extensions/wikia/MercuryApi/handlers/MercuryApiFilePageHandler.class.php

```diff
@@ -12,8 +12,25 @@ public static function getFileContent( Title $title ) {
 		$details = WikiaFileHelper::getMediaDetail( $title );
 		$mediaObject = ArticleAsJson::createMediaObject( $details, $title->getText() );
 
+		// if article contains user provided HTML which is invalid (e.g. to many </div>), snippetter treat it as a text
```

Comment:
'too'

Reply:
done line below

Reply:
I think they meant `e.g. to many </div>` -> `e.g. too many </div>`